docs/src/main/paradox/docs/delta/api/current/views/elasticsearch-view-api.md

```diff
@@ -2,17 +2,19 @@
 
 This view creates an ElasticSearch `index` and stores the targeted Json resources into an ElasticSearch Document.
 
-The documents created on each view are isolated from documents created on other views by using different ElasticSearch indices.
+The documents created on each view are isolated from documents created on other views by using different ElasticSearch
+indices.
 
 A default view gets automatically created when the project is created but other views can be created.
 
 ## Processing pipeline
 
-An asynchronous process gets trigger for every view. This process can be visualized as a pipeline with different stages. 
+An asynchronous process gets trigger for every view. This process can be visualized as a pipeline with different stages.
 
 The first stage is the input of the pipeline: a stream of events scoped for the project where the view was created.
 
-The last stage takes the JSON document, generated through the pipeline steps, and stores it as a Document in an ElasticSearch index
+The last stage takes the JSON document, generated through the pipeline steps, and stores it as a Document in an
+ElasticSearch index
 
 [![ElasticSearchView pipeline](../assets/views/elasticsearch_pipeline.png "ElasticSearchView pipeline")](../assets/views/elasticsearch_pipeline.png)
```

Comment:
the image you posted in the PR description does not show this picture... maybe the path is wrong?